QUEST_JOBSTEP.tsv

```diff
@@ -1311,13 +1311,13 @@ QUEST_JOBSTEP_20150317_001310	Go to the Archer Master in Klaipeda.
 QUEST_JOBSTEP_20150317_001311	Explore the area the Archer Master mentioned
 QUEST_JOBSTEP_20150317_001312	The Archer Master is looking for a book and he says he'll take you as an Archer if you help him find the book. Search the Vubbe Outpost and check if the you can find the book there.
 QUEST_JOBSTEP_20150317_001313	Deliver book to Master Archer
-QUEST_JOBSTEP_20150317_001314	Found the book the Archer Master is looking for. Take it to the Archer Master.
-QUEST_JOBSTEP_20150317_001315	This isn't the only book the Archer Master is looking for. Talk to the Archer Master again.
-QUEST_JOBSTEP_20150317_001316	The Hunter Master has the book the Archer Master wants. Prepare 5000 silvers to get the book from the Hunter Master.
-QUEST_JOBSTEP_20150317_001317	Gathered enough money for the book. Give 5000 silvers to the Hunter Master in exchange for the book and take the book to the Archer Master.
-QUEST_JOBSTEP_20150317_001318	The Archer Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001319	Collect the Merrogg Leather
-QUEST_JOBSTEP_20150317_001320	The Archer Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather you have for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
+QUEST_JOBSTEP_20150317_001314	Found the book Archer Master is looking for. Take it to Archer Master.
```

Comment:
Found the book Archer Master is looking for. Take it to Archer Master. -> Found the book that the Archer Master is looking for. Take it to the Archer Master.